Tests/BaseTestCase.php

```diff
@@ -48,6 +44,11 @@ public function tearDown()
         $this->schemaTool->dropSchema($this->getSchemaClasses());
     }
 
+    public function getConfig()
+    {
+        return Yaml::parse(file_get_contents(__DIR__ . '/../config/database.yml'));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'getConfig'.](https://www.codacy.com/app/Managlea/EntityManager/file/2304164333/issues/source?bid=3149133&fileBranchId=3194846#l49)